src/commands/SetClipCommand.cpp

```diff
@@ -95,7 +95,7 @@ bool SetClipCommand::ApplyInner( const CommandContext &, Track * t )
                pClip->SetColourIndex(mColour);
             // No validation of overlap yet.  We assume the user is sensible!
```

Comment:
Notice this comment.  The non-overlapping condition for clips can be violated with macro programming!  Not our responsibility to fix that yet I think.